snuba/cli/devserver.py

```diff
@@ -54,6 +54,17 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 "--consumer-group=transactions_group",
             ],
         ),
+        (
+            "sessions-consumer",
+            [
+                "snuba",
+                "consumer",
+                "--auto-offset-reset=latest",
```

Comment:
well I guess we can do this for devserver, otherwise we would have to run this in error mode which is the default.